tests/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set(STL_SOURCE_DIR "${PROJECT_SOURCE_DIR}")
+set(STL_BUILD_ROOT "${PROJECT_BINARY_DIR}/out")

```

Comment:
this may be wrong. If you want the directory into which we are putting binaries you just need PROJECT_BINARY_DIR

Reply:
No, this is right considering the state of the root `CMakeLists.txt`. All the output directories are relative to `${PROJECT_BINARY_DIR}/out`. I don't particularly think that that is the right thing to do but this conforms to that.